### PR TITLE
sdgPlots

### PR DESCRIPTION
**Pull request by lilyclements**
Edited sdgPlots
ucrAdditionalLayer
ucrAxes

Fixed bugs

### CODE REVIEW

File: instat/sdgPlots.vb [hunk 1]
```diff
@@ -13,6 +13,7 @@
 '
 ' You should have received a copy of the GNU General Public License k
 ' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+Imports instat
```
Comment (dannyparsons) [inline instat/sdgPlots.vb]: What is this needed for?

File: instat/sdgPlots.vb [hunk 2]
```diff
@@ -331,11 +336,11 @@ Public Class sdgPlots
     End Sub
 
     Private Sub chkNoOfRowsOrColumns_CheckedChanged(sender As Object, e As EventArgs) Handles chkNoOfRowsOrColumns.CheckedChanged
-        nudNoOfRowsOrColumns.Visible = chkNoOfRowsOrColumns.Checked
+        nudNumberofRows.Visible = chkNoOfRowsOrColumns.Checked
         SetFacets()
     End Sub
 
-    Private Sub nudNoOfRowsOrColumns_TextChanged(sender As Object, e As EventArgs) Handles nudNoOfRowsOrColumns.TextChanged
+    Private Sub nudNumberofRows_ValueChanged(sender As Object, e As EventArgs) Handles nudNumberofRows.ValueChanged
```
Comment (dannyparsons) [inline instat/sdgPlots.vb]: We can keep this as TextChanged

File: instat/ucrAxes.vb [hunk 3]
```diff
@@ -49,6 +49,14 @@ Public Class ucrAxes
...
         nudUpperLimit.Value = 0
         nudScalesNoOfDecimalPlaces.Value = 0
+        lblFrom.Visible = False
+        lblTo.Visible = False
+        lblInStepsOf.Visible = False
+        lblTickMarkersNoOfDecimalPlaces.Visible = False
+        nudFrom.Visible = False
+        nudTo.Visible = False
+        nudInStepsOf.Visible = False
+        nudTickMarkersNoOfDecimalPlaces.Visible = False
```
Comment (dannyparsons) [inline instat/ucrAxes.vb]: Visibility shouldn't be a default. It should only be controlled only by methods. Maybe those methods need to run in SetDefaults?

**Comment by lilyclements** [comment]
Removed ImportsInstat (not sure why that added in but I've noticed it keeps adding in in a few places)
Made .TextChanged
Removed defaults that were added - realised the issue is coming from elsewhere

**Pull request merged**